### PR TITLE
Gidk 2x8wrw gst stage responsive1

### PR DESCRIPTION
**Pull request by shalinee-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/invoice/preview/models/download-or-send-mail/download-or-send-mail.component.html [hunk 1]
```diff
@@ -82,4 +82,80 @@ <h3 class="font-16 mr-b05">{{localeData?.invoice_copy}}</h3>
                 [src]="sanitizedPdfFileUrl"></iframe>
         </div>
     </div>
+
+
+    <!--  -->
+    <div class="modal-header clearfix" *ngIf="isMobileScreen">
+        <span>
+            Download PDF
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/invoice/preview/models/download-or-send-mail/download-or-send-mail.component.html:90]: Avoid hard coded labels

File: apps/web-giddh/src/app/invoice/preview/models/download-or-send-mail/download-or-send-mail.component.html [hunk 2]
```diff
@@ -82,4 +82,80 @@ <h3 class="font-16 mr-b05">{{localeData?.invoice_copy}}</h3>
...
+        </span>
+        <i class="icon-cross text-right close-modal" aria-hidden="true" (click)="onCancel('closed')"></i>
+        <!-- drop down for email area -->
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/invoice/preview/models/download-or-send-mail/download-or-send-mail.component.html:93]: Remove commented code

File: apps/web-giddh/src/app/invoice/preview/models/download-or-send-mail/download-or-send-mail.component.html [hunk 3]
```diff
@@ -82,4 +82,80 @@ <h3 class="font-16 mr-b05">{{localeData?.invoice_copy}}</h3>
...
+                <button type="button" class="btn btn-md btn-success mr-b2" (click)="onSendInvoiceOnMail(emailList.value)">Send</button>
+            </div>
+        </div> -->
+
+
+    </div>
+    <div class="modal-body  p-0 clearfix" id="export-body" *ngIf="isMobileScreen">
+
+        <!-- drop down for invoice copies -->
+
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/invoice/preview/models/download-or-send-mail/download-or-send-mail.component.html:108]: Avoid too many empty lines

**Review by prateek-giddh — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/gst/filing/tabs/overview/view-transactions/view-transactions.component.html [hunk 4]
```diff
@@ -184,8 +184,10 @@
                 <tbody>
                     <tr class="cursor-pointer" *ngFor="let item of (viewTransaction$ | async).results">
                         <td>{{item.voucherDate ? item.voucherDate : '-'}}</td>
-                        <td class="btn-link" *ngIf="item.voucherType !== 'purchase'" (click)="onSelectInvoice(item)">
+                        <td class="btn-link" *ngIf="(item.voucherType !== 'purchase')" (click)="onSelectInvoice(item)">
                             {{item.voucherNumber ? item.voucherNumber : '-'}}
+
+                            <span *ngIf="isMobileScreen" class="d-inline-flex text-decoration-none btn-link"><i class="icon-download-circle font-16 ml-2"></i></span>
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/gst/filing/tabs/overview/view-transactions/view-transactions.component.html:190]: Please check there is no click listener added to this mobile screen HTML

**Review by prateek-giddh — CHANGES_REQUESTED** [review]

**Review by ravinderwalkover — APPROVED** [review]

File: apps/web-giddh/src/app/invoice/preview/models/download-or-send-mail/download-or-send-mail.component.html (hunk 1 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/invoice/preview/models/download-or-send-mail/download-or-send-mail.component.html]: Done

File: apps/web-giddh/src/app/invoice/preview/models/download-or-send-mail/download-or-send-mail.component.html (hunk 3 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/invoice/preview/models/download-or-send-mail/download-or-send-mail.component.html]: Done

File: apps/web-giddh/src/app/invoice/preview/models/download-or-send-mail/download-or-send-mail.component.html (hunk 2 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/invoice/preview/models/download-or-send-mail/download-or-send-mail.component.html]: Done

File: apps/web-giddh/src/app/gst/filing/tabs/overview/view-transactions/view-transactions.component.html (hunk 4 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/gst/filing/tabs/overview/view-transactions/view-transactions.component.html:190]: Click even added on upper td

**Review by prateek-giddh — APPROVED** [review]

**Review by aditi-tiwari17 — APPROVED** [review]

**Pull request merged**